docs/architecture/pairing-adapter-for-integration-system.md

```diff
@@ -0,0 +1,29 @@
+# Pairing Adapter for Integration System
+
+## Introduction
+
+When dealing with multiple Integration Systems each of them may support different ways of establishing trusted relation for the given Application instance. To provide a consistent experience for users, we introduce the concept of Pairing Adapters. Each Integration System can be coupled with one Pairing Adapter which will abstract the details on how the Integration System handles the trusted relation prerequisites.
+
+## Solution overview
+
+This diagram illustrates the flow for pairing the Application with its External Broker using the Pairing Adapter.
+
+![External Broker Pairing Adapter](./assets/external-broker-pairing-adapter.svg)
```

Comment:
- rename External Broker to `Integration System Instance`. That change will add more consistency.
- after point 6, add point 7 and solid line between `Integration System Instance` and Application.